src/QsCompiler/Tests.DocGenerator/DocParsingTests.cs

```diff
@@ -352,5 +352,20 @@ ArgDeclType BuildArgument(string name, ResolvedType t)
             var s = stream.ToString();
             Assert.Equal(expected, s);
         }
+
+        [Fact]
+        public void ParseDeprecated()
+        {
+            string[] comments = { "# Summary",
```

Comment:
Currently, deprecated warnings for our docs are generated based on a section in the doc comments  
/// # Deprecated
/// Please use @"Microsoft.Quantum.NewName".
It would be good to add a test if the docs generation is well behaved if both are present.